tools/Makefile.unix

```diff
@@ -308,7 +308,7 @@ ifneq ($(CONFIG_ARCH_CHIP),)
 	$(Q) touch $@
 endif
 
-dirlinks: include/arch include/arch/board include/arch/chip $(ARCH_SRC)/board $(ARCH_SRC)/chip drivers/platform
+dirlinks: tools/incdir$(HOSTEXEEXT) include/arch include/arch/board include/arch/chip $(ARCH_SRC)/board $(ARCH_SRC)/chip drivers/platform
```

Comment:
Should `incdir` be removed from pass1/2 targets now?

Reply:
why?  It still needs to be built.  The change to Config.mk doesn't build it.  It just prevents using it if it is not built.

I don't understand what would happen if you remove the dependency from pass1 and pass1.  I think 'make pass1 pass2' would then fail, correct?  Or it would just be very slow because the incdir binary was never built.

I believe the build would fail without these.  But you should give it a try.

'make pass1 pass2' is actually recommended in some README.txt files.

Reply:
It just won't be needed because it would've already been built during dirlinks (which is also a part of context)